### ISSUE TITLE
Docker Desktop does not start, and uninstall hangs

### ISSUE THREAD

**Issue by afeldmansbux** [docker/for-win#5617]
I am unable to start Docker Desktop as of approximately 1/23.  Docker attempts to start until it times out and pops up an error message.
```
One or more errors occurred.

   at Docker.Core.Pipe.NamedPipeClient.<TrySendAsync>d__5.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at Docker.Core.Pipe.NamedPipeClient.Send(String action, Object[] parameters)
   at Docker.Actions.DoStart(SynchronizationContext syncCtx, Boolean showWelcomeWindow, Boolean withNotifications)
   at Docker.Actions.<>c__DisplayClass26_0.<StartAsync>b__0()
   at Docker.ApiServices.TaskQueuing.TaskQueue.<>c__DisplayClass18_0.<.ctor>b__1()
```

I am also unable to uninstall Docker Desktop.  The uninstaller hangs at the removing VMs step.  Uninstalling Hyper-V allowed me to finally uninstall Docker Desktop, however reinstalling both Docker and Hyper-V puts me right back where I started.

This repros with Docker Desktop v 2.2.0.0 and Edge 2.1.7.0.  I am running Windows 10 
1709.
[log.zip](https://github.com/docker/for-win/files/4120126/log.zip)

**Comment by marcselis** [comment]
Same problem here, also on Windows 10 1709.

**Comment by afeldmansbux** [comment]
Updating to Windows 10 1809 seems to have fixed it for me.  Seems there may be incompatibility between the latest Docker Desktop releases and older Windows 10 versions.

**Comment by marcselis** [comment]
Not an option for me.  My PC is managed by my company.

**Comment by afeldmansbux** [comment]
Same for me, however the Update section in Settings did have an option to bypass the Enterprise update service and check online for updates with Microsoft.  You might give that a try.

**Comment by marcselis** [comment]
I don't have that option, but I fixed it by revering back to version 2.1.0.5 and disabling update check for now.
